README.md

```diff
@@ -149,11 +149,11 @@ If the command succeeds you should get a result like this:
    - **Client Secret** ( Azure Active Directory -> App Registrations -> Choose your app -> Keys)
 
 5. Paste these three items into the fields in the Azure Data Explorer API Details section:
-   ![Azure Data Explorer API Details](https://raw.githubusercontent.com/grafana/azure-data-explorer-datasource/master/src/img/config_2_azure_data_explorer_api_details.png)
+   ![Azure Data Explorer API Details](https://raw.githubusercontent.com/grafana/azure-data-explorer-datasource/main/src/img/config_2_azure_data_explorer_api_details.png)
 
 6. Click the `Save & Test` button. After a few seconds once Grafana has successfully connected then choose the default database and save again.
 
-[on-behalf-of documentation](/doc/on-behalf-of.md)
+[on-behalf-of documentation](https://github.com/grafana/azure-data-explorer-datasource/blob/main/doc/on-behalf-of.md)
```

Comment:
random thought unrelated to this pr: I wonder if we should add some kind of linting rule or something to be like "hey don't use relative links to markdown files" because I could totally see myself doing that in the future not thinking about how it would break some sort of automated doc link lol

Reply:
are there any linting rules that check for relative links? might be useful for the markdown files that get published like README, etc.